test/check-machines-create

```diff
@@ -931,6 +931,8 @@ class TestMachinesCreate(VirtualMachinesCase):
             if self.sourceTypeSecondChoice:
                 b.select_from_dropdown("#source-type", self.sourceTypeSecondChoice)
 
+            # Data loading doesn't start immediately
+            time.sleep(1)
             b.wait_attr("#os-select-group", "data-loading", "false")
             # For the mocked fedora URL installation wait to see that the OS section is automatically filled
```

Comment:
Just a question, but with this sleep, are we waiting for OS data to be loaded? I thought that's what the following              `b.wait_attr("#os-select-group", "data-loading", "false")` does? Or maybe it's broken?

Reply:
This sleep waits for the loading to _start_.  The following `wait_attr` then waits for the loading to be finished.

If you look at the current pixel test reference in main, you can see that the test catches the dialog with the OS field disabled, which means that the data is currently being loaded, no?

![image](https://user-images.githubusercontent.com/3228183/151817330-17b2241b-f132-4bb1-8573-dda1854aaf61.png)

Reply:
Ah right I see. It seems it's a result of how `onOsAutodetect` sets `autodetectOSInProgress` value. Perhaps that could be somehow rewritten so there is no sleep, but that's outside of this PR.

Reply:
Ah right I see. It seems it's a result of how onOsAutodetect sets autodetectOSInProgress value. Perhaps that could be somehow rewritten so there is no sleep, but that's outside of this PR.